src/FSharp.Control.TaskSeq/TaskSeq.fsi

```diff
@@ -844,7 +844,7 @@ type TaskSeq =
     static member takeWhileAsync: predicate: ('T -> #Task<bool>) -> source: TaskSeq<'T> -> TaskSeq<'T>
 
     /// <summary>
-    /// Returns a sequence that, when iterated, yields elements of the underlying sequence until the given
+    /// Returns a task sequence that, when iterated, yields elements of the underlying sequence until the given
```

Comment:
Maybe this contraction does not lose the meaning? the commas make it all read very legalese?
```suggestion
    /// Returns a task sequence yielding elements of the underlying sequence until the given
```

Reply:
Yeah, I was mulling over this as well. I took it from `Seq.takeWhile` originally, I believe. But there's nothing that stops us from improving on that.

I used your change in all variants where it made sense.

Reply:
Actually, while doing this, I ran into the "making it less clear for other functions" issue (like `init`), and I like to use the same language where possible. 

For now, I stick with it. We can certainly overhaul it at some point in the near future.